lib/galaxy/webapps/galaxy/buildapp.py

```diff
@@ -271,6 +271,7 @@ def app_pair(global_conf, load_app_kwds=None, wsgi_preflight=True, **kwargs):
     webapp.add_client_route("/interactivetool_entry_points/list")
     webapp.add_client_route("/libraries{path:.*?}")
     webapp.add_client_route("/storage{path:.*?}")
+    webapp.add_client_route("/headings_demo")
```

Comment:
Where is this route handled?

Reply:
It was temporary, for a test -- just needs to be deleted.
```suggestion
```